File: src/main/java/io/github/dylmeadows/eontimer/model/timer/CustomTimerModel.java
package io.github.dylmeadows.eontimer.model.timer;

import javafx.collections.FXCollections;
import javafx.collections.ObservableList;
import lombok.Data;

import java.time.Duration;

@Data
public class CustomTimerModel {

    private final ObservableList<Long> stages = FXCollections.observableArrayList();
}

File: src/main/kotlin/io/github/dylmeadows/eontimer/model/timer/CustomTimerModel.kt
package io.github.dylmeadows.eontimer.model.timer

import io.github.dylmeadows.eontimer.model.Stage
import javafx.collections.FXCollections
import javafx.collections.ObservableList

class CustomTimerModel {
    val stages: ObservableList<Stage> = FXCollections.observableArrayList<Stage>()
}